### PR TITLE
Request for UNOCS ontology

### PR DESCRIPTION
**Pull request by SebSeis**
This is a request for the new unocs ontology

### CODE REVIEW

**Comment by dgarijo** [comment]
@SebSeis you are changing the ownership of the TAO ontology, I am not sure this is what you intend.
Also, you are adding a redirect for `^ocqa`, but maybe you want `unocs`? (or remove the `^ocqa` bit)

**Comment by SebSeis** [comment]
Yeah. You are right. I fixed the "ocqa" in unocs htaccess file. I copied wrong. But i dont know why this tao ontology appears.... I have to fix this.

**Comment by dgarijo** [comment]
See https://github.com/perma-id/w3id.org/issues/4417, it may be related to that

**Comment by SebSeis** [comment]
Okay. I changed the files for Tao back using the original files. (Found it here https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request) So, the changes should now only affect the UNOCS ontology.

File: unocs/htaccess.txt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ https://sebseis.github.io/UNOCS/ [R=302,NE,L]
+
+# In case of accept header <text/turtle>
+RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.* 
+RewriteRule ^$ https://sebseis.github.io/UNOCS/unocs.ttl [R=303,NE,L]
+
+# If suffix ttl, redirect to turtle version
+RewriteRule ^uncos.ttl$ https://sebseis.github.io/UNOCS/unocs.ttl [R=302,NE,L]
```
Comment (dgarijo) [inline unocs/htaccess.txt:21]: I think you want unocs, not uncos? Same for html

Comment (SebSeis) [reply unocs/htaccess.txt:21]: Yeah. Sry. I don't know what's going on today...

**Comment by dgarijo** [comment]
@SebSeis a few more comments on your latest PR. Please have a look

**Comment by SebSeis** [comment]
Thanks for your support!

**Pull request merged**